content/faq/app/how-to-use-android-auto/index.it.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Come si usa Android Auto?
+description: "Domande frequenti sull'applicazione Organic Maps".
+
```

Comment:
```suggestion
slug: come-si-usa-android-auto
```

Reply:
Yup, wasn't sure about the slug. Thanks!